commit 49da8e8c5376c16d92a892d939d0283678ba0717 (refs/changes/80/3380/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-15T17:40:27-08:00 (1 year, 8 months ago)
    
    TRITON-139 Triton's config-agents should poll at 120s interval as a stopgap (update default pollInterval)

diff --git a/agent.js b/agent.js
index 0996195..25736f8 100644
--- a/agent.js
+++ b/agent.js
@@ -68,7 +68,7 @@ if (configPath) {
 	config = {
 		instances: [],
 		logLevel: 'info',
-		pollInterval: 60000
+		pollInterval: 120000
 	};
 }
 
